resource_control: RU info in explain analyze (#14234)
pingcap/docs: sql-statements/sql-statement-explain-analyze.md

- `write_keys`: The total `keys` written in the transaction.
- `write_byte`: The total bytes of `key-value` written in the transaction, and the unit is byte.

### RU (Request Unit) Consumption

> [Request Unit (RU)](/tidb-resource-control.md#what-is-request-unit-ru) a unified abstraction unit of system resources which is defined in TiDB resource control. The `execution info` of top level operator shows the overall RU consumption of this particular SQL statement.

Reviewer: ```suggestion
[Request Unit (RU)](/tidb-resource-control.md#what-is-request-unit-ru) is a unified abstraction unit of system resources which is defined in TiDB resource control. The `execution info` of top level operator shows the overall RU consumption of this particular SQL statement.
```

```
RU:273.842670
```

> > **Note :**

Reviewer: ```suggestion
> **Note:**
```

>

> > This value shows the actual RU produced by this call. The RU consumption could vary if the same SQL statement repeats, as the amount of RU is affected by cache layer (for example, [coprocessor cache](/coprocessor-cache.md)).

Reviewer: ```suggestion
> This value shows the actual RU produced by this call. The RU consumption might vary if the same SQL statement repeats, as the amount of RU is affected by cache layer (for example, [coprocessor cache](/coprocessor-cache.md)).
```

> > This value shows the actual RU produced by this call. The RU consumption might vary if the same SQL statement repeats, as the amount of RU is affected by cache layer (for example, [coprocessor cache](/coprocessor-cache.md)).

Reviewer: ```suggestion
> This value shows the actual RU consumed by this execution. The same SQL might consume different number of RUs each time it is executed due to the effects of caching (for example, [coprocessor cache](/coprocessor-cache.md)).
```

> > This value shows the actual RU consumed by this execution. The same SQL might consume different amounts of RUs each time it is executed due to the effects of caching (for example, [coprocessor cache](/coprocessor-cache.md)).

Reviewer 2: ```suggestion
> This value shows the actual RUs consumed by this execution. The same SQL statement might consume different amounts of RUs each time it is executed due to the effects of caching (for example, [coprocessor cache](/coprocessor-cache.md)).
```
